lib/Semantics/resolve-names.cpp

```diff
@@ -5169,6 +5213,12 @@ void ConstructVisitor::SetTypeFromAssociation(Symbol &symbol) {
   }
 }
 
+// Set the rank of symbol based on the current rankCase value extracted from
```

Comment:
What is the purpose of this function?  Why not call `Symbol::SetRank()` directly?

Reply:
Ok, I was trying to create something more readable similar to `SetTypeFromAssociation` and `SetAttrsFromAssociation`,